commerce/domain/pricing/service/classes/PricingServiceSample.cls

```diff
@@ -141,6 +145,49 @@ public class PricingServiceSample extends commercestorepricing.PricingService {
     return txnResponse;
   }
 
+  private Map<String, PriceDataFromExternalService> getPriceFromStaticResponse(Set<String> skus) {
```

Comment:
This was because of the comment in this file which mentioned
// If the request is successful, parse the JSON response; The response includes the negotiated
// as well as list price for each SKU. It looks something like this: response =
// {"ProductID1":{"listPrice":10,"unitPrice":5}};

If this is meant to be IDs we can modify this @jwaref

Reply:
"The response" in this comment refers to the response from heroku service. What this means is that the static data should be mapped by sku but the input to the method should be product Ids. Does that make sense?

Reply:
resolved